modules/sdk-coin-zketh/test/unit/send.ts

```diff
@@ -0,0 +1,79 @@
+import { TransactionType } from '@bitgo/sdk-core';
+import should from 'should';
+import { decodeTransferData } from '@bitgo/abstract-eth';
```

Comment:
I think this needs to be inside transactionBuilder folder

Reply:
done